const todoOk = nombre => console.log("Todo genial 游봃. Eres increible " + nombre);
const algoFallo = error => console.error("Algo salio mal 游땰", error);

// const funcionPromesa = (cumplio, noCumplio) => {
//     if (false) {
//         cumplio("Omar");
//     } else {
//         noCumplio("La condici칩n est치 en false");
//     }
// };

// const miPrimerPromesa = new Promise((resolve, reject) => {
//     if (false) {
//         resolve("Omar");
//     } else {
//         reject("La condici칩n est치 en false");
//     }
// });
// miPrimerPromesa.then(todoOk);
// miPrimerPromesa.catch(algoFallo);

const miPromesa = () => {
    return new Promise((resolve, reject) => {
        if (true) resolve("Omar");
        else reject("La condici칩n est치 en false");
    });
};

miPromesa()
.then(nombre => console.log("Todo genial 游봃. Eres increible " + nombre))
.catch(error => console.error("Algo salio mal 游땰", error));